api/net/Areas/Subscriber/Models/UserModel.cs

```diff
@@ -0,0 +1,157 @@
+using System.Text.Json;
+using TNO.API.Models;
+
+namespace TNO.API.Areas.Subscriber.Models;
+
+/// <summary>
+/// UserModel class, provides a model that represents an user.
+/// </summary>
+public class UserModel : AuditColumnsModel
+{
+    #region Properties
+    /// <summary>
+    /// get/set - Primary key to user.
+    /// </summary>
+    public int Id { get; set; } = default!;
+
+    /// <summary>
+    /// get/set - Unique key to identify the user.
+    /// </summary>
+    public string Key { get; set; } = "";
+
+    /// <summary>
+    /// get/set - Unique username to identify user.
+    /// </summary>
+    public string Username { get; set; } = "";
+
+    /// <summary>
+    /// get/set - User's email address.
+    /// </summary>
+    public string Email { get; set; } = "";
+
+    /// <summary>
+    /// get/set - Display name of user.
+    /// </summary>
+    public string DisplayName { get; set; } = "";
+
+    /// <summary>
+    /// get/set - First name of user.
+    /// </summary>
+    public string FirstName { get; set; } = "";
+
+    /// <summary>
+    /// get/set - Last name of user.
+    /// </summary>
+    public string LastName { get; set; } = "";
+
+    /// <summary>
+    /// get/set - Whether the user is enabled.
+    /// </summary>
+    public bool IsEnabled { get; set; }
+
+    /// <summary>
+    /// get/set - The user status.
+    /// </summary>
+    public Entities.UserStatus Status { get; set; }
+
+    /// <summary>
+    /// get/set - Whether the user email is verified.
+    /// </summary>
+    public bool EmailVerified { get; set; }
+
+    /// <summary>
+    /// get/set - Whether the user is a system account.
+    /// </summary>
+    public bool IsSystemAccount { get; set; }
```

Comment:
Do not include any fields they should not be able to see or edit (IsEnabled, Status, EmailVerified, IsSystemAccount, LastLoginOn, Note, Roles).  Otherwise you're introducing security holes.

Reply:
This is why we use different models for each endpoint.